web/modules/custom/joinup_search/config/install/facets.facet.type.yml

```diff
@@ -11,7 +11,7 @@ name: Type
 url_alias: type
 weight: 0
 min_count: 1
-show_only_one_result: true
+show_only_one_result: false
```

Comment:
We need to test that multiple choices can be selected. Probably you can do that in `tests/features/joinup_search/search.feature`.

Reply:
I'm on it!

Reply:
Test added to multiple choices and broken tests also have been fixed.